metrics-jmx/src/main/java/com/codahale/metrics/jmx/JmxReporter.java

```diff
@@ -648,6 +652,7 @@ public void onTimerAdded(String name, Timer timer) {
             } catch (JMException e) {
                 LOGGER.warn("Unable to register timer", e);
             }
+            return null;
```

Comment:
Why don't we return the `Timer` which has been passed into the method when it was added?